pmg/views.py

```diff
@@ -491,10 +491,50 @@ def attendance_overview():
     Display overview of attendance for meetings.
     """
     this_year = datetime.today().year
-    attendance = CommitteeMeetingAttendance.annual_attendance_trends(
-        to_year=this_year)
+    attendance = CommitteeMeetingAttendance.annual_attendance_trends(to_year=this_year)
```

Comment:
We had a bug which led to something not working when we were confident it would and told the client it's done.

Manual inspection wasn't sufficient here - let's add the automated testing needed to be confident we won't need to talk to the client about this being broken again.

Reply:
The issue was that we had not implemented the attendance for the current data since it was a bit tricky dealing with the upcoming 6 months.

Reply:
There's all this code that isn't tested?